### ISSUE TITLE
Drone 0.5 agent doesn't connect to server

### ISSUE THREAD

**Issue by MorrisJobke** [drone/drone#1672]
I use a Debian 8 and run drone/drone:0.5 via:

```
$ docker run --volume /var/lib/drone:/var/lib/drone --env-file /etc/dronerc --restart=always --publish=80:8000 --detach=true --name=drone drone/drone:0.5
```

With following dronerc:

```
DRONE_GITHUB=true
DRONE_GITHUB_CLIENT={{ github_client_id }}
DRONE_GITHUB_SECRET={{ github_client_secret }}
DRONE_OPEN=true
DRONE_ADMIN={{ drone_admins }}
DRONE_ORGS={{ drone_orgs }}
DRONE_SECRET={{ drone_secret }}
GIN_MODE=release
DEBUG=true
I_UNDERSTAND_I_AM_USING_AN_UNSTABLE_VERSION=true
I_AGREE_TO_FIX_BUGS_AND_NOT_FILE_BUGS=true
```

(Yes I put the `I_AGREE_TO_FIX_BUGS_AND_NOT_FILE_BUGS` in there, but nevertheless file a bug to have this documented because on gitter it's quite hard to follow the discussion)

The output of this container in the logs is:

```
time="2016-06-13T07:35:33Z" level=warning msg="agents can connect with token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0ZXh0IjoiWWE2YWlzYWl0aGlpemVpdDVEZTJlaXBoNmVpQmVpMmwiLCJ0eXBlIjoiYWdlbnQifQ.ti1glEEuN2bftZyJ-vNWWfR24QHtBK1o_EUNC5SkC8s" 
```

I used this to connect the agent via following command:

```
$ docker run --volume /var/run/docker.sock:/var/run/docker.sock -e DRONE_SERVER=http://192.168.0.28 --restart=always --detach=true --name=drone-agent -e DRONE_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0ZXh0IjoiWWE2YWlzYWl0aGlpemVpdDVEZTJlaXBoNmVpQmVpMmwiLCJ0eXBlIjoiYWdlbnQifQ.ti1glEEuN2bftZyJ-vNWWfR24QHtBK1o_EUNC5SkC8s" -e DRONE_DEBUG=true drone/drone:0.5 agent
```

The output is then (even after minutes of waiting):

```
time="2016-06-13T07:37:40Z" level=info msg="Connecting to http://192.168.0.28 with token eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0ZXh0IjoiWWE2YWlzYWl0aGlpemVpdDVEZTJlaXBoNmVpQmVpMmwiLCJ0eXBlIjoiYWdlbnQifQ.ti1glEEuN2bftZyJ-vNWWfR24QHtBK1o_EUNC5SkC8s" 
```

@bradrydzewski Told me, that then something in the logs should appear that indicates that a agent has connected to the server but I haven't seen this.

Also the IP could be reached from outside of the virtual machine - then the login page is shown. If pinged from the host itself it is reachable. 

I used following container (pulled 1h ago):

```
$ docker pull drone/drone:0.5
0.5: Pulling from drone/drone
a3ed95caeb02: Already exists 
802d894958a2: Already exists 
3dfd2983f122: Pull complete 
Digest: sha256:85b59ee0d5fa741785a4b86049a5bb6d1b70e9af6388a73f6033287385bc1ce0
Status: Downloaded newer image for drone/drone:0.5
```

How could I help to debug this further?

**Comment by bradrydzewski** [comment]
closing because `I_AGREE_TO_FIX_BUGS_AND_NOT_FILE_BUGS=true`
